packages/terra-form-radio/src/RadioField.jsx

```diff
@@ -109,13 +113,17 @@ const RadioField = (props) => {
     legendAttrs.className,
   ]);
 
+  const feildSetId = `terra-radio-group-${uniqueid()}`;
   const legendAriaDescriptionId = `terra-radio-field-description-${uniqueid()}`;
   const helpAriaDescriptionId = help ? `terra-radio-field-description-help-${uniqueid()}` : '';
   const errorAriaDescriptionId = error ? `terra-radio-field-description-error-${uniqueid()}` : '';
   const ariaDescriptionIds = `${legendAriaDescriptionId} ${errorAriaDescriptionId} ${helpAriaDescriptionId}`;
 
+  const isSafariOREdge = ((navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1) || navigator.userAgent.indexOf('Edg') !== -1);
+  const Component = (isSafariOREdge) ? 'div' : 'legend';
```

Comment:
`Component` is too vague of a variable name; can we name it to something else like `LegendGroup`?
```suggestion
  const Component = (isSafariOREdge) ? 'div' : 'legend';
```

Reply:
i have made changes as per suggestions